src/core/model_config_utils.cc

```diff
@@ -998,7 +998,8 @@ ValidateModelConfig(
         "'max_batch_size' must be non-negative value for " + config.name());
   }
 
-  if (!config.platform().empty()) {
+  //If backend is  empty, validate the platform
```

Comment:
explain why it is not needed if backend is set, something like "Otherwise, the platform validation should be performed by the backend if necessary as Triton may not know about the backend's requirements"

Reply:
Good catch. Added the explanation.